packages/design-system-twrnc-preset/src/Theme/Theme.utilities.ts

```diff
@@ -83,6 +74,5 @@ export const generateTailwindConfig = (
         },
       },
     },
-    plugins: baseConfig.plugins || [],
```

Comment:
This is an optional type in the `TwConfig` since we aren't using it, it can be removed